packages/lab/src/calendar/Calendar.tsx

```diff
@@ -73,7 +73,7 @@ export const Calendar = forwardRef<HTMLDivElement, CalendarProps>(
         <div
           className={clsx(withBaseName(), className)}
           role="application"
-          aria-label={calendarLabel}
+          aria-label={`Calendar, ${calendarLabel}`}
```

Comment:
Can we move "Calendar" to the modal overlay

Reply:
you mean out of calendar and in date picker instead?

Reply:
Yes please